### DISCUSSION TITLE
Offer expanded DOI resolution statistics

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#105] [General Suggestions]
### What is the problem that your suggestion solves?

Repositories are not able to see reports on all DOI resolutions for their DOIs. A repository would like to answer questions like what is our most popular dataset this year, how many times has a particular DOI been resolved, and how have resolution rates changed over time.

### What solution might meet your needs?

Access to up-to-date DOI resolutions statistics for all DOIs. 

### What alternatives have you tried or considered?

Using [DataCite Statistics](https://stats.datacite.org/resolutions.html#tab-resolution-report) to check top 10 DOI successes. 

### Is there anything else you would like to share?

[Link for DataCite staff](https://docs.google.com/spreadsheets/d/16jQnuW_DMgbb-74Y5TByUG4IkHdvJSZ0Fe_qxQ4Xl6w/edit?gid=444055379#gid=444055379&range=127:127)

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
A member noted that they would like to retrieve resolution statistics by DOI by month.

[Link for DataCite staff](https://app.frontapp.com/open/cnv_b9k75xx?key=tOdwp6xNZII4-1PhKqi55hmpzcvM6Naa)

**Comment by jezcope** [comment]
Thanks Cody, for a bit more info here is what I wrote in my original email to DataCite support. Hope it helps!

In the first instance, raw logs from CNRI that have been filtered down to the given prefix(es) would be something I could work with, though I appreciate the logistics might be tricky as those are probably rather large...

-----

The resolution report can be quite useful, but for each 1-month period it shows aggregate stats for the whole prefix and individual stats only for the top 10 DOIs that period.

Some things that I would like to be able to do that aren't possible with this format:

- Plot a month-by-month time series for a given DOI over a longer period (unless that DOI is in the top 10 for every month in the period, and then it's necessary to go through month-by-month on the web page and copy/paste the numbers, or scrape the page)
- Identify top 10 DOIs over a longer time period by adding up their monthly stats
- Aggregate statistics for subsets of DOIs within the prefix (e.g. grouping by record type)
- Identify longer-term patterns in DOI resolution failures

The exact format wouldn't matter, but something shaped like this would work:

| DOI                | Status  | Period  | Count |
|--------------------|---------|---------|-------|
| 10.23636/rcm4-zk44 | Success | 2024-01 | 270   |
| 10.23636/rcm4-zk44 | Success | 2024-02 | 222   |
| …                  | …       | …       | …     |
| 10.23636/5t9m-0g59 | Success | 2024-01 | 318   |
| …                  | …       | …       | …     |
| 10.23636/toq3-vn04 | Failure | 2024-01 | 7     |
| …                  | …       | …       | …     |

**Comment by ej2432** [comment]
Having enhanced Resolution Reporting would be really helpful for me as well, as a DataCite member. I get a monthly resolution report from Crossref which includes all of the successful and failed DOI attempts for the past month. This allows for me to troubleshoot and remediate issues related to the DOIs so that the problems are resolved or explained to future users. 

At the minimum, having an option to export "Resolutions by Month" in CSV would be great.

Even expanding this to the top 100 successful and unsuccessful DOIs would go a long way for my needs.
